Merge Astro and Software CLI docs (#1317)
astronomer/docs: astro/cli/astro-deployment-create.md

title: "astro deployment create"
id: astro-deployment-create
description: Reference documentation for astro deployment create.
hide_table_of_contents: true
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

:::info  

> The behavior and format of this command differs depending on what Astronomer product you're using. Use the following tabs to change between product contexts. 

Reviewer: ```suggestion
The behavior and format of this command differs depending on what Astronomer product you're using. Use the following tabs to change product contexts. 
```

| `-v`,`--runtime-version`    | The Astro Runtime version for the Deployment                                                                                   | Any supported version of Astro Runtime. Major, minor, and patch versions must be specified. |
| `-s`,`--scheduler-au`       | The number of AU to allocate towards the Deployment's Scheduler(s). The default is`5`.                                         | Integer between `0` and `24`                                                                |
| `-a`,`--worker-au`          | The number of AU to allocate towards the Deployment's worker(s). The default is `10`.                                          | Integer between `0` and `175`                                                               |
| `-r`,`--scheduler-replicas` | The number of scheduler replicas for the Deployment. The default is `1`.                                                       | Integer between `0` and `4`                                                                 |

> | `--wait`                    | Waits until the new Deployment has a [healthy](deployment-metrics.md#deployment-health) status before completing the command . | None                                                                                        |

Reviewer: ```suggestion
| `--wait`                    | The time to wait for the new Deployment to have a [healthy](deployment-metrics.md#deployment-health) status before completing the command . | None                                                                                        |
```

# Specify 50 AU for the Deployment's workers. The Astro CLI prompts you for required information
```

</TabItem>

<TabItem value="software">

Create a Deployment on Astronomer Software. This command is functionally identical to using the Software UI to create a Deployment.

## Usage

```sh
astro deployment create
```

## Options

| Option                        | Description                                                                                                                                                        | Possible Values                                                                                                                      |
| ----------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------ |

> | `-a`,`--airflow-version`      | The version of Astronomer Certified to use for the Deployment                                                                                                      | Any supported version of Astronomer Certified                                                                                        |  |

Reviewer: ```suggestion
| `-a`,`--airflow-version`      | The Astronomer Certified version to use for the Deployment                                                                                                      | Any supported version of Astronomer Certified                                                                                        |  |
```
